srcpkgs/sane-airscan/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'sane-airscan'
+pkgname=sane-airscan
+version=0.99.24
+revision=1
+wrksrc="sane-airscan-${version}"
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="avahi-libs-devel sane-devel gnutls-devel libxml2-devel libjpeg-turbo-devel libpng-devel"
+short_desc="SANE driver for Apple AirScan"
+maintainer="pudiva <rogi@skylittlesystem.org>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/alexpevzner/sane-airscan"
+distfiles="http://github.com/alexpevzner/sane-airscan/archive/${version}.tar.gz"
+checksum=723ec0a139943d268615082d76ab860d7aa24a5d2b53763f49ecec4293b3d934
+
+do_install() {
+	: ${make_cmd:=make}
+	: ${make_install_target:=install}
```

Comment:
It's okay to hardcode these in the command. What you might be able to do is add `make_install_args="STRIP=echo"` or use some conditionals to get the correct `strip` for cross and native compilation there.

Reply:
@ericonr, thanks for the suggestion! Indeed I could get around defining a custom `do_install()` by setting `$make_install_args="STRIP=-s"`...

However, Makefile uses `$(STRIP)` as options to `install`:
https://github.com/alexpevzner/sane-airscan/blob/master/Makefile#L118-L132

and defaults to `-s` (which instructs `install` to strip):
https://github.com/alexpevzner/sane-airscan/blob/master/Makefile#L30-L35

so, for cross-compilation, I think we would need to override `$(INSTALL)` with `target-linux-gnu-install` ~ which could be a nice addition to the gnu-makefile build style, as it already does many similar overrides for `CC`, `LD` and friends.

What do you think?

Reply:
It's actually `install --strip-program=target-linux-gnu-strip`. Would be a bit strange to have platform-specific `install`...

Reply:
We can do `STRIP=-s`, because we have an `install` wrapper that removes the flag anyway :p So you don't need to care about passing the correct `--strip-program` :)

We do the stripping ourselves, to generate debug packages. I quite dislike build systems that try to strip binaries themselves, tbh.